vstools/utils/scale.py

```diff
@@ -186,9 +176,9 @@ def get_lowest_values(
     ], fmt.num_planes)
 
 
-def get_neutral_value(clip_or_depth: int | VideoFormatT | HoldsVideoFormatT, chroma: bool = False) -> float:
+def get_neutral_value(clip_or_depth: int | VideoFormatT | HoldsVideoFormatT) -> float:
     """
-    Returns the mid point value for the specified bit depth, or bit depth of the clip/format specified.
+    Returns the neutral point value for the specified bit depth, or bit depth of the clip/format specified.
```

Comment:
I think this would make the meaning significantly clearer:

```suggestion
    Returns the neutral point value (e.g. as used by std.MakeDiff) for the specified bit depth, or bit depth of the clip/format specified.
```